travis/test_precommit.py

```diff
@@ -52,14 +57,43 @@ def copy_cfg_files(cfg_dirname, repo_dirname):
                 fdst.write(line)
 
 
+def disable_env_checks(conf_path):
+    conf_path = os.path.join(os.getcwd(), conf_path)
+    args_str = f"--disable={disabled_checks}"
```

Comment:
Using f-string we will be compatible MQT only for `>=py3.6` and we even have old projects using py2.7

And pylint, pylint-odoo and MQT has multi-compatibility

Reply:
Yes, I changed it to `"--disable=%s" % disabled_checks`